qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -838,6 +846,7 @@ def main():
         # convert into absolute path
         options.script = os.path.abspath(os.path.expanduser(options.script))
         if not os.path.exists(options.script):
+            # TODO should be logged
```

Comment:
This has been duplicated?

Reply:
Good catch. I must have slipped on the keyboard and pressed copy line!